okhttp-brotli/src/test/java/okhttp3/brotli/BrotliInterceptorTest.kt

```diff
@@ -13,6 +13,9 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:Suppress(
+  "INVISIBLE_REFERENCE",
+)
```

Comment:
This creates a warning
<img width="1051" height="105" alt="image" src="https://github.com/user-attachments/assets/290c8c8b-652d-4d38-96e6-7051232e2473" />

And the IDE is suggesting that I add `ERROR_SUPPRESSION` but it doesn't make the warning go away

Reply:
@swankjesse found this, it's an IDE only bug with handling of Kotlin friends. So unavoidable?

Reply:
It doesn’t bother me.